### DISCUSSION TITLE
Create Pull Request form mobile app

### DISCUSSION THREAD

**Discussion by TWiStErRob** [community/community#5734] [Mobile]
I can create an issue and work with existing PRs, but there's no way to open a new one.

**Comment by alcere** [comment]
Hi @TWiStErRob!
You are right, right now right now the apps don't allow creating a Pull request. I'm going to include your feature request to our internal tracking issue.

**Reply by davidenke** [reply]
Any updates about this? It'll be nice create pull requests from existing branches.

**Comment by aconsuegra** [comment]
Hi everyone! We have added the possibility to create pull requests from existing branches on GitHub Mobile. It's available in version 1.193 and you can read more about the announcement in https://github.blog/changelog/2025-01-22-create-pull-request-from-an-existing-branch-on-github-mobile/ 

Let us know if you have any feedback!
